department-of-veterans-affairs/caseflow
# Update intake form benefit types for updated VA Form 20-0995 and 20-0996

Resolves [CASEFLOW-2260](https://vajira.max.gov/browse/CASEFLOW-2260) and [CASEFLOW-2261](https://vajira.max.gov/browse/CASEFLOW-2261) 

### Description

- Update Supplemental Claim intake form to use '"Veterans Readiness and Employment" instead of "Vocational Rehabilitation and Employment"
- Update Higher Level Review intake form to use "Veterans Readiness and Employment" instead of "Vocational Rehabilitation and Employment"
- Update Higher Level Review intake form to use "Veterans Readiness and Employment" instead of "Vocational Rehabilitation and Employment"
-Update Higher Level Review intake form to not use the question “Was an interview by the same office requested?”

### Acceptance Criteria
- [ ] Update behind feature toggle

### Testing Plan
1. Enable feature toggle `veterans_readiness`
2. Switch to intake user
3. Select Supplemental Claim as form type
4. Search for a Veteran
5. Verify benefit type option "Veterans Readiness and Employment" exists *instead of* "Vocational Rehabilitation and Employment"

### User Facing Changes

 BEFORE|AFTER
 --------|-------
<img width="1072" alt="Screen Shot 2021-11-03 at 9 53 37 PM" src="https://user-images.githubusercontent.com/68879427/140260406-5cdf1dbd-91d2-42f6-bc58-202d7b2e9559.png">|<img width="832" alt="Screen Shot 2021-11-03 at 12 44 31 PM" src="https://user-images.githubusercontent.com/68879427/140260417-608f4a94-e864-4963-bf6a-0c9a2dc2bc29.png">|---
<img width="957" alt="Screen Shot 2021-11-05 at 10 27 25 AM" src="https://user-images.githubusercontent.com/68879427/140553444-40669820-9599-4abc-8286-f406a1132005.png">|<img width="966" alt="Screen Shot 2021-11-05 at 10 25 07 AM" src="https://user-images.githubusercontent.com/68879427/140553473-7db31e13-523a-42c6-a824-a133572c6d1e.png">
